src/virtualenv/create/via_global_ref/builtin/python2/site.py

```diff
@@ -52,8 +52,7 @@ def load_host_site():
     add_site_dir = sys.modules["site"].addsitedir
     for path in json.loads(site_packages):
         full_path = os.path.abspath(os.path.join(here, path.encode("utf-8")))
-        if full_path not in sys.path:
-            add_site_dir(full_path)
+        add_site_dir(full_path)
```

Comment:
This feels wrong to me. The path here is the last resort to ensure these paths are on the sys.path. Can we find out why the site package reload does not puts the python path onto the sys path and load the site-packages?

Reply:
@gaborbernat, I'm not sure I understand. In commit 1e869a97 you adjusted exactly this place to make sure pth files are processed _at_ _all_. That was on Ubuntu and I'm on Debian. Python maintainer for Ubuntu and Debian is the same person, so those packages are likely to have the same properties. Are you suggesting to me to find out why pth files were not processed before 1e869a97 and find a "more proper" fix for the problem?